### PR TITLE
In 135431629 add slaves

### PR DESCRIPTION
**Pull request by ThomasOSI**
@pemaOSI ready for review.
Note: This will be followed by changes the intake and intake_api_prototype repos to set a label on their nodes.

### CODE REVIEW

File: deploy-jenkins.yml [hunk 1]
```diff
@@ -3,6 +3,16 @@
     src: ./docker-compose.jenkins.yml.j2
     dest: '{{ project_source }}/docker-compose.jenkins.yml'
 
+- name: Copy Makefile for slaves
+  copy:
+    src: ./Makefile
```
Comment (pemaOSI) [inline deploy-jenkins.yml]: Can we organize files under proper folders such as jenkins related files could go under jenkins folder etc.

**Comment by pemaOSI** [comment]
Left comments but this repo need proper organization. We have files related api, jenkins and intake app

**Comment by ThomasOSI** [comment]
@pemaOSI Reorganization has been completed and tested.

**Comment by pemaOSI** [comment]
If you tested it out, then it looks good.

**Pull request merged**